### DISCUSSION TITLE
Release 2.1.1

### DISCUSSION THREAD

**Discussion by jkuester** [jkuester/unlauncher#212] [General]
## Features
* Add popup menu for apps in the drawer by @ru2saig in https://github.com/jkuester/unlauncher/pull/134
* Adds search field options by @hayribakici in https://github.com/jkuester/unlauncher/pull/143
* Start first app by pressing Enter in search field by @khwolf in https://github.com/jkuester/unlauncher/pull/158
* Prioritize apps starting with query by @khwolf in https://github.com/jkuester/unlauncher/pull/165
* Add icons to popup menu for the main apps by @khwolf in https://github.com/jkuester/unlauncher/pull/166
* Update Simplified Chinese Strings by @sr093906 in https://github.com/jkuester/unlauncher/pull/156
* Completes German translation by @khwolf in https://github.com/jkuester/unlauncher/pull/160
* Adds option to choose the alignment of apps by @khwolf in https://github.com/jkuester/unlauncher/pull/161
* Group apps in app drawer by first letter by @hayribakici in https://github.com/jkuester/unlauncher/pull/144
* Add new themes: Darcula and Gruvbox by @hayribakici in https://github.com/jkuester/unlauncher/pull/138
* Full-width clickability by @khwolf in https://github.com/jkuester/unlauncher/pull/178
* Update Simplified Chinese Strings by @sr093906 in https://github.com/jkuester/unlauncher/pull/177
* Russian translation update by @zepp in https://github.com/jkuester/unlauncher/pull/171
* Make hidden apps findable with search by @JuanJakobo in https://github.com/jkuester/unlauncher/pull/130
* Add titles to configuration views by @khwolf in https://github.com/jkuester/unlauncher/pull/181
* Update Simplified Chinese Strings by @sr093906 in https://github.com/jkuester/unlauncher/pull/184
* Update German translation by @khwolf in https://github.com/jkuester/unlauncher/pull/185
* Add different kinds of clocks by @khwolf in https://github.com/jkuester/unlauncher/pull/186
* Update Simplified Chinese Strings by @sr093906 in https://github.com/jkuester/unlauncher/pull/197
* Update French strings by @milouse in https://github.com/jkuester/unlauncher/pull/211

## Bug fixes

* Use correct text color in options switches by @hayribakici in https://github.com/jkuester/unlauncher/pull/179
* Correct case insensitive sorting by @khwolf in https://github.com/jkuester/unlauncher/pull/180
* Remove uninstalled apps from home screen by @hayribakici in https://github.com/jkuester/unlauncher/pull/164
* Fix clock when search box at bottom by @jkuester in https://github.com/jkuester/unlauncher/pull/193
* Update app drawer whenever a change was triggered by @jkuester in https://github.com/jkuester/unlauncher/pull/194
* Fix header color on Customise Drawer page by @hayribakici in https://github.com/jkuester/unlauncher/pull/195
* Handle boxed W for work apps in drawer header by @jkuester in https://github.com/jkuester/unlauncher/pull/199
* Hide uninstall option if app is system app. by @hayribakici in https://github.com/jkuester/unlauncher/pull/198
* Use alignment option in app drawer by @TobiTenno in https://github.com/jkuester/unlauncher/pull/200

## Chores

* Update DEVELOPMENT.md by @fusaty in https://github.com/jkuester/unlauncher/pull/176
* Unify font sizes by @khwolf in https://github.com/jkuester/unlauncher/pull/154
* Cean up and sort translation files by @jkuester in https://github.com/jkuester/unlauncher/pull/192
* Update docs regarding use of the Proto datastore by @jkuester in https://github.com/jkuester/unlauncher/pull/204
* Use dimension for most layout distances by @khwolf in https://github.com/jkuester/unlauncher/pull/201

## New Contributors
* @ru2saig made their first contribution in https://github.com/jkuester/unlauncher/pull/134
* @fusaty made their first contribution in https://github.com/jkuester/unlauncher/pull/176
* @zepp made their first contribution in https://github.com/jkuester/unlauncher/pull/171
* @JuanJakobo made their first contribution in https://github.com/jkuester/unlauncher/pull/130
* @milouse made their first contribution in https://github.com/jkuester/unlauncher/pull/211

<hr /><em>This discussion was created from the release <a href='https://github.com/jkuester/unlauncher/releases/tag/2.1.1'>Release 2.1.1</a>.</em>

**Comment by Albeiro403** [comment]
Muchas gracias 
Thanks you
